vllm/v1/engine/processor.py

```diff
@@ -108,15 +108,11 @@ def process_inputs(
 
         # Make Request for Detokenizer.
         detokenizer_request = DetokenizerRequest(
-            request_id,
-            decoder_inputs.prompt,
-            decoder_inputs.prompt_token_ids,
+            request_id, decoder_inputs.prompt, decoder_inputs.prompt_token_ids,
             sampling_params.skip_special_tokens,
             sampling_params.spaces_between_special_tokens,
-            sampling_params.output_kind,
-            sampling_params.stop,
-            sampling_params.include_stop_str_in_output,
-        )
+            sampling_params.output_kind, sampling_params.stop,
+            sampling_params.include_stop_str_in_output, lora_request)
```

Comment:
nit: Please add `,` at the end and rerun the formatter so that other lines don't change. 
```suggestion
            sampling_params.include_stop_str_in_output, lora_request,
            )
```